src/test/java/task40/DownloadPercentageTest.java

```diff
@@ -0,0 +1,38 @@
+package task40;
```

Comment:
Please create one more package/folder in task40 with name 'test' and move all your test classes to it